src/main/java/com/amazon/aocagent/services/EC2Service.java

```diff
@@ -69,7 +71,7 @@ public EC2Service(String region) {
    * @param amiID the instance amiid
    * @return InstanceID
    */
-  public Instance launchInstance(String amiID) throws Exception {
+  public Instance launchInstance(String amiID, boolean isARM) throws Exception {
```

Comment:
maybe you can use TestAMI as the parameter instead of the string-amid, so that you don't need to have a "isARM"

Reply:
yes that a good point I can do that